doc/guides/publishing-napi-modules.md

```diff
@@ -0,0 +1,27 @@
+## To publish an N-API version of your package alongside your non-N-API version
+The following steps are illustrated using the package `iotivity-node`:
+  - First, publish the non-N-API version:
+    - Update the version in your `package.json`. For `iotivity-node`, the version becomes `1.2.0-2`.
```

Comment:
We have always treated guides as being less formal in style, but I've been wanting to move away from that, starting with avoiding the use of personal pronouns (`you` and `your`, etc) in the documentation.